src/a.tex

```diff
@@ -195,6 +193,11 @@ \section{Load-Reserved/Store-Conditional Instructions}
 A platform specification may constrain the size and shape of the reservation
 set.  For example, the Unix platform is expected to require the set be
 contiguous and no greater than the virtual memory page size.
+
+In the common case where only aligned LR/SC is supported and each
+reservation set covers one or multiple of the largest supported LR/SC,
+each reservation covers exactly one reservation set.  Otherwise,
+a reservation covers one or more (whole) reservation sets.
```

Comment:
I would suggest strengthening the premise.  Something like, "In the standard A extension, only aligned LR/SC is supported, and so each reservation covers exactly one reservation set."

Reply:
The standard extension wording works.  It also makes sense for reservation sets to have the same alignment restrictions as LR/SC, I'll add that.  Independently, reservation sets might still be smaller than 8 bytes -- a doubleword LR/SC might cover two.  So the statement isn't so straightforward.  Perhaps:

> In the standard A extension, only aligned LR/SC is supported:  assuming each reservation set covers the largest LR/SC, each reservation covers exactly one reservation set.  Otherwise, a reservation covers one or more (whole) reservation sets.